frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -153,6 +153,7 @@ end
 function ReaderRolling:onReaderReady()
     self:setupTouchZones()
     self.setupXpointer()
+    self.old_page = 0
```

Comment:
In that case you could just check for  

if self.old_page then

I think being undefined should suffice? No need to initialize to 0 and check if larger.